docs/site/HasMany-relation.md

```diff
@@ -178,6 +180,55 @@ export interface OrderRelations {
 export type OrderWithRelations = Order & OrderRelations;
 ```
 
+### Relation Metadata
+
+`keyFrom`, `keyTo` and `name` fields in the `hasMany` relation matadata allow us
+to customize and specify our id/foreign key names:
+
+- `keyFrom`: the primary key of the source model. The default value is the id
+  property in the source model, which is `id` property of `Customer` in our
+  example.
+- `keyTo`: the foreign key of the target model. The default value is the source
+  model name appended with `id` in camel case, which is the `customerId`
+  property of `Order` in our example.
+- `name`: the name of the relation, which is the `orders` property of `Customer`
+  in our example.
+
+We can specify the `keyTo` field via `@hasMany` decorator if we have customized
+the foreign key name as `my_customer_id` instead of `customerId`:
+
+```ts
+class Customer extends Entity {
+  // constructor, properties, etc.
+  @hasMany(() => Order, {keyTo: 'my_custom_id'})
+  orders: Order[];
+}
+```
+
+```ts
+@model()
+export class Order extends Entity {
+  // constructor, properties, etc.
+  @property({
+    type: 'number',
+  })
+  my_customer_id: number; // customized foreign key name
+}
+```
+
+Notice that if you decorate the foreign key of the target model with
+`@belongsTo`, you also need to specify the relation name in its metadata. See
+[BelongsTo](BelongsTo-relation.md) for more details.
```

Comment:
Should this link to the relation metadata part of BelongsTo?

Reply:
There is no certain section in belongsTo

Reply:
BelongsTo doesn't have such a section.